Readme.md

```diff
@@ -53,11 +57,67 @@ Callback URL needs to be added to the Authentication tab as a Web Redirect URI
 
 **search_api_url \<OSDU Host Here\>/api/search/v2/query**
 
-**data-partition-id opendes**
+**data-partition-id  \<Your Data Partiton ID Here\>**
 
 These parameters need to be provided by the deployment team for deployments in
 customer tenants
 
+## Authorization and Entitlements
```

Comment:
Instead of documenting how to add a user to an entitlement group, can you link to a community OSDU doc to do it?